source/Calamari.Tests/Fixtures/PackageDownload/PackageDownloadFixture.cs

```diff
@@ -99,9 +99,6 @@ public void ShouldDownloadPackage()
         [RequiresNonFreeBSDPlatform]
         public void ShouldDownloadMavenPackage()
         {
-            if (CalamariEnvironment.IsRunningOnMac && TestEnvironment.IsCI && !CalamariEnvironment.IsRunningOnMono)
```

Comment:
These whole inconclusive tests seem a bit weird. 
We should be able to remove this condition entirely now, but if the tests play up ill take a look at readjusting.

Reply:
Seems like these haven't busted up any tests - nice removal!